packages/smarthr-ui/src/libs/ua.ts

```diff
@@ -1,20 +1,28 @@
 const ua = typeof window !== 'undefined' ? window.navigator.userAgent.toLowerCase() : 'SSR'
 
+const isWindows = ua.indexOf('windows') !== -1
```

Comment:
nits) めっちゃ細かいんですが、includesとか使ったほうがすっきりしますかね？
```suggestion
const isWindows = ua.includes('window')
```

Reply:
ちょっともともとなぜindexOfを使っていたのかがわからないのであれ何ですが、正規表現化しちゃうの有りだと思います。
ただ今回のやつはあくまでiOS系対応になるのでやるとしても別PRにしたいです

Reply:
はい！ちょっと筋違いなこと言ってるな...とは思っておりましたので賛成です :+1: